examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -72,7 +128,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Solve using the configuration with QTip"
+    "## Solving using the quantum SDK QuTiP"
```

Comment:
Could you detail what this means for our users?

Reply:
Done (a bit better)